scanner/describe/describe.go

```diff
@@ -177,7 +177,10 @@ func (s *Scanner) parseLine(b []byte) Line {
 	// Looking for double space, as some keys have spaces in them, e.g:
 	// "QoS Class:                   Burstable"
 	//            ^endOfKey
-	endOfKey := bytesutil.IndexOfDoubleSpace(leftTrimmed)
+	endOfKey := bytesutil.IndexOfEqualOrColon(leftTrimmed)
+	if endOfKey < 0 {
+		endOfKey = bytesutil.IndexOfDoubleSpace(leftTrimmed)
+	}
```

Comment:
I suggest removing the labels parsing out from this `scanner/describe` package. This package does not handle nested values.

The way it parsers labels is that all the labels lines are treated as values.

For example, an input like this:

```
Labels:           app=traefik
                  team=platform
```

Should be parsed as:

- Line 1:
  - Key: `Labels:`
  - Value: `app=traefik`
- Line 2:
  - Key: ` `
  - Value: `team=platform`

So then if you want additional parsing and coloring, I'd say it's better to add that into the `printer/kubectl_describe.go` file, as a custom logic when the "path" is `"Labels"`

Look at how the `describeUseStatusColoring` finds if a given value should use `ColorStatus`. You could do a similar approach but then do your `string.CutString(s, "=")` logic and color the label key-value pairs explicitly.